.github/workflows/test.yml

```diff
@@ -18,14 +18,26 @@ jobs:
           submodules: "recursive"
 
       - name: Add msbuild to PATH
-        uses: microsoft/setup-msbuild@v1
+        uses: microsoft/setup-msbuild@v2
 
-      - name: Build Projects
+      - name: Build CLEO core
         shell: cmd
         run: |
           set PLUGIN_SDK_DIR=%GITHUB_WORKSPACE%\third-party\plugin-sdk
           msbuild -m CLEO5.sln /property:Configuration=Release /property:Platform=GTASA
+          
+      - name: Build CLEO plugins
+        shell: cmd
+        run: |
+          set PLUGIN_SDK_DIR=%GITHUB_WORKSPACE%\third-party\plugin-sdk
           msbuild -m cleo_plugins/CLEO_Plugins.sln /property:Configuration=Release /property:Platform=x86
+          
+#      - name: VirusTotal Scan
+#        uses: crazy-max/ghaction-virustotal@v4
+#        with:
+#          vt_api_key: ${{ secrets.VT_KEY }}
+#          files: |
+#            ./.output/Release/CLEO.asi
```

Comment:
@MiranDMC can you keep this step and send CLEO.asi and all plugins to test? I think it can be useful for future detections

Reply:
Reenabled